test/web-platform-tests/to-run.yaml

```diff
@@ -466,14 +470,9 @@ reset-form.html: [fail, Unknown]
 DIR: html/semantics/forms/the-button-element
 
 button-activate.html: [fail, Unknown]
-button-checkvalidity.html: [fail, Validation is not implemented]
 button-events.html: [fail, Unknown]
 button-labels.html: [fail, labels attribute is not implemented]
-button-setcustomvalidity.html: [fail, Validation is not implemented]
-button-validation.html: [fail, Validation is not implemented]
-button-validationmessage.html: [fail, Validation is not implemented]
-button-validity.html: [fail, Validation is not implemented]
-button-willvalidate.html: [fail, Validation is not implemented]
+button-validation.html: [fail, Expects type 'menu' to not change to 'submit']
```

Comment:
Sounds like this test is broken upstream; it'd be ideal to file a bug or send a PR to fix it. (Not a blocker for this PR.)

Reply:
It was a while ago, so I may not be recalling it well, I think it was jsdom at fault here as it's the behaviour that both Chrome and Firefox exhibited.